src/garage/torch/policies/discrete_qf_argmax_policy.py

```diff
@@ -65,4 +65,4 @@ def get_actions(self, observations):
             dict: Empty since this policy does not produce a distribution.
         """
         with torch.no_grad():
-            return self(torch.Tensor(observations)).numpy(), dict()
+            return self(torch.Tensor(observations)).cpu().numpy(), dict()
```

Comment:
how do you i know i'm sampling on CPU?

Reply:
Well regardless of where the model is the output tensor would need to be moved to the CPU to convert it to a numpy array (the `numpy()` call isn't supported for cuda tensors.

Reply:
@krzentner can you comment here? not sure what to do in torch

Reply:
Yeah, we expect policies to always output numpy arrays. However, instead of using `torch.Tensor`, you should probably use `garage.torch.np_to_torch`.